### ISSUE TITLE
genxq option broken

### ISSUE THREAD

**Issue by gzheng92** [mdroste/stata-binscatter2#13]
This occurs on Stata 17 with updated `gtools`

MWE:
```
set obs 10000
gen x = rnormal()
gen y = 5* x + rnormal()
binscatter2 y x, nograph genxq(x_bin)
```

yields a return code of 111, for "variable doesn't exist"

repeating the same code using `binscatter` in lieu of `binscatter2` produces the expected bins

**Comment by gzheng92** [comment]
After further testing, even omitting the options on `binscatter2` yields a r(111).  I think it's an issue with gtools

**Comment by mdroste** [comment]
Fixed in version 0.24!
